docs/mindsdb-docs/mkdocs.yml

```diff
@@ -90,8 +90,8 @@ nav:
   - Upload a File to MindsDB: sql/create/file.md
   - Train a Predictor: sql/create/predictor.md
   - Get a Prediction: sql/api/select.md
+  - MindsDB Tables Structure: sql/table-structure.md
```

Comment:
I would personally put this back into the SQL section.

Reply:
Done.